src/styles/getContextMenuItemStyles/types.ts

```diff
@@ -0,0 +1,5 @@
+import {ViewStyle} from 'react-native';
+
+type GetContextMenuItemStyle = (windowWidth?: number) => ViewStyle[];
```

Comment:
Any reason why `windowWidth` might undefined?

Reply:
I see one usage of `GetContextMenuItemStyle` with an argument:
https://github.com/Expensify/App/blob/5d45f0f6c86e97d4ae58ac167b6c162a610b817c/src/components/ContextMenuItem.js#L100

Reply:
This is actually something that comes from [TS guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#file-organization). It was made that way to keep types consistent for native as well as other platforms. 😄